### DISCUSSION TITLE
貌似又更新了啥策略，自己搭建的代理坏掉了

### DISCUSSION THREAD

**Discussion by xiaoguaishou92** [linweiyuan/go-chatgpt-api#264] [Q&A]
ERRO[0001] Failed to start, please try again later. 
WARN[0001] 
    body{font-family:Arial,Helvetica,sans-serif}.container{align-items:center;display:flex;flex-direction:column;gap:2rem;height:100%;justify-content:center;width:100%}@keyframes enlarge-appear{0%{opacity:0;transform:scale(75%) rotate(-90deg)}to{opacity:1;transform:scale(100%) rotate(0deg)}}.logo{color:#8e8ea0}.scale-appear{animation:enlarge-appear .4s ease-out}@media (min-width:768px){.scale-appear{height:48px;width:48px}}.data:empty{display:none}.data{border-radius:5px;color:#8e8ea0;text-align:center}@media (prefers-color-scheme:dark){body{background-color:#343541}.logo{color:#acacbe}}

**Comment by linweiyuan** [comment]
没有改什么，只是加了个日志，全部输出内容只有这些？

这个错误是因为官网不稳定，重试或者过段时间再试就好了

**Reply by xiaoguaishou92** [reply]
还有很多，下面的有token，没贴。
 Please stand by, while we are checking your browser...Redirecting...<h1 style="color:#bd2426;">Please turn JavaScript on and reload the page.</h1>Please enable Cookies and reload the page.(function(){window._cf_chl_opt={cvId: '2',cZone: "chat.openai.com",cType: 'managed',cNounce: '3163',cRay: '80bee09afa3c1d8f',cHash: 'd34b845ca4564c6',cUPMDTk: "\/backend-api\/accounts\/check?__cf_chl_tk=C26VS_eaPv9lsJoA1CPKHcF88h0YCBpH7gz8oS01x4M-1695598877-0-gaNycGzNCvs",cFPWv: 'g',cTTimeMs: '1000',cMTimeMs: '0',cTplV: 1,cTplB: 'cf',cK: "visitor-time",fa: "\/backend-api\/accounts\/check?__cf_chl_f_tk=C26VS_eaPv9lsJoA1CPKHcF88h0YCBpH7gz8oS01x4M-1695598877-0-gaNycGzNCvs",md:

**Reply by linweiyuan** [reply]
这个启动不需要 token 吧？

回头升级下依赖包试试

**Reply by linweiyuan** [reply]
昨天升级了依赖，试试，再不行提供你的代理给我调试，PROXY 地址

**Reply by xiaoguaishou92** [reply]
set proxy to: socks5://xxxx:65535
<nil>
set proxy to: socks5://xxxx:65535

可以用了，我看启动日志多了这两行打印~

**Reply by linweiyuan** [reply]
今晚看看，应该是打多了
